dashboard/test/controllers/concerns/pd/workshop_filters_test.rb

```diff
@@ -68,10 +66,12 @@ class FakeController < ::ApplicationController
     expects(:not).with(course: Pd::Workshop::COURSE_CSF)
 
     params course: '-csf'
-    @controller.load_filtered_ended_workshops
+    load_filtered_ended_workshops
   end
 
   test 'filter_workshops default' do
```

Comment:
In general you're verifying that the filters you expect to be applied are actually being applied, but not asserting on the final output. Expected?